decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -88,6 +88,8 @@ def install_decidim_webpacker
 
         # Create empty directory for images
         empty_directory "app/packs/images"
```

Comment:
As far as I see, this is the only place where the `empty_directory` method is used.